lib/UtapiReindex.js

```diff
@@ -61,6 +61,8 @@ class UtapiReindex {
             this._log = new werelogs.Logger('UtapiReindex', { level, dump });
         }
 
+        this._onlyCountLatestWhenObjectLocked = (config && config.onlyCountLatestWhenObjectLocked !== undefined);
```

Comment:
This looks strange to me, shouldn't we not activate the option if `config.onlyCountLatestWhenObjectLocked` is `false`?